rtos/mbed_rtos_rtx1_types.h

```diff
@@ -0,0 +1,30 @@
+/* mbed Microcontroller Library
```

Comment:
are these types from rtx1 or cmsis-rtos 1 (based on that the file should be named). I think we were using CMSIS-RTOS API and its types.